File: fibonacci.cpp
#include <iostream>
#include <string>
#include <cstdlib>
using namespace std;

int main() {
  int num1{1}, num2{1}, length{10}, sum{0};

  for (int i = length; i > 0; i--) {
    sum = num1 + num2;
    num1 = num2;
    num2 = sum;
  }

  string stringSum = to_string(sum);
  cout << "Digit: " << sum << endl;
  cout << "last char: " << stringSum.at(stringSum.length() - 1) << endl;

  return 0;
}

File: fibonacci.js
const main = (num1, num2, length) => {
  let n1 = num1,
    n2 = num2,
    sum = 0;

  for (let i = length; i > 0; i--) {
    sum = n1 + n2;
    n1 = n2;
    n2 = sum;
  }

  let stringSum = sum.toString();
  console.log(stringSum[stringSum.length - 1]);
  // 1, 3, 5, 8, 13, 21, 34, 55, 89, 144
};

main(1, 1, 10);
